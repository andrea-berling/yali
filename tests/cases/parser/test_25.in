(-49 + 27) * (57 * 16) / (48 + 63)
