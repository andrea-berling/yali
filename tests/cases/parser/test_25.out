(/ (* (group (+ (- 49.0) 27.0)) (group (* 57.0 16.0))) (group (+ 48.0 63.0)))
